services/QuillLMS/client/app/bundles/Shared/libs/translations/helpers.tsx

```diff
@@ -18,13 +18,17 @@ interface renderSaveAndExitArguments {
   translate: (language: string) => string
 }
 
+function handleClick() {
+  window.location.href = `${window.postbuildEnv.DEFAULT_URL}/profile`
+}
+
 export function renderSaveAndExitButton({ isELLDiagnostic, language, languageOptions, translate }: renderSaveAndExitArguments) {
   const languages = isELLDiagnostic ? defaultLanguages : languageOptions?.map(option => option.value)
   let buttonText = 'Save and exit'
   if (language && language !== ENGLISH && languages && languages.includes(language)) {
     buttonText = translate('buttons^save and exit')
   }
-  return <a className="quill-button medium contained white focus-on-dark" href={`${window.postbuildEnv.DEFAULT_URL}/profile`}>{buttonText}</a>
```

Comment:
Why did this get changed? I think it's more accessible as a an `<a />` tag (because it'll be announced as a link to screenreader users).

Reply:
I was getting an error in ARIA dev tools and thought a button would be more accessible, but I can change it back. I think it just needs a role (which should be `role="link"` right?)-- just an aside, I think maybe that error just presents locally and not on staging or in prod. I'm realizing that now